TestingTool/tests/register_email_dsc_required.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "test 'email is required' err on dsc front end register",
+  "url": "",
+  "mobile_mode": true,
+  "tasks": [
+    {
+      "name": "Go to dsc frontend",
+      "action": "go_to_url",
+      "args": {
+        "url": "http://79.112.76.0:32774/"
+      }
+    },
+    {
+      "name": "click register",
+      "action": "click_element",
+      "args": {
+        "xpath": "/html/body/app-root/div/app-welcome-page/div/div[2]/button[1]"
+      }
+    },
+    {
+      "name": "click email",
+      "action": "click_element",
+      "args": {
+        "xpath": "//*[@id=\"mat-input-2\"]"
+      }
+    },
+    {
+      "name": "click first name field (random element from webpage)",
```

Comment:
Mai scoate te rog din pasi, sunt inutili si doar 'consuma timp' fara nici-un rost :))
E nevoie de un 'proof of concept' minimal doar

Reply:
am scos si am facut un nou pr